composer.json

```diff
@@ -2,47 +2,53 @@
     "type": "project",
     "license": "proprietary",
     "require": {
-        "php": "^7.4.1 || ^8.0.0",
+        "php": ">=8.0.2",
         "ext-ctype": "*",
         "ext-iconv": "*",
         "composer/package-versions-deprecated": "^1.11",
         "doctrine/doctrine-bundle": "^2.1",
         "doctrine/doctrine-migrations-bundle": "^3.0",
         "doctrine/orm": "^2.7",
-        "knplabs/knp-markdown-bundle": "^1.8",
+        "knplabs/knp-markdown-bundle": "dev-symfony6",
```

Comment:
Nice job - I assume you created a PR for this to the bundle?

Reply:
I didn't... that bundle already has Sf6-related PR, but that one is a bit more complex than just composer constraints